source/Irrlicht/CMakeLists.txt

```diff
@@ -142,7 +144,7 @@ endif()
 # OpenGL
 
 if(USE_SDL2)
-	option(ENABLE_OPENGL3 "Enable OpenGL 3+" TRUE)
+	option(ENABLE_OPENGL3 "Enable OpenGL 3+" FALSE)
```

Comment:
rebase mistake?

Reply:
I temporarily disabled the new driver to make CI pass: 69622fe641017a10eaf05d0f36186fea424d80ec
I'll revert that commit and rebase this PR now that #250 is merged.